static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -131,13 +148,33 @@ function DefaultGroupEventDetailsContent({
         />
       </StyledDataSection>
       {event.userReport && (
-        <EventDataSection title={t('User Feedback')} type="user-feedback">
-          <EventUserFeedback
-            report={event.userReport}
-            orgSlug={organization.slug}
-            issueId={group.id}
-            showEventLink={false}
-          />
+        <EventDataSection
+          title={t('User Feedback')}
+          type="user-feedback"
+          actions={
+            <ErrorBoundary mini>
+              <ButtonBar gap={1}>
+                <HideUserFeedbackButton
+                  isLoading={promptLoading}
+                  isError={promptError || isPromptDismissed === undefined}
+                  isHidden={isPromptDismissed !== undefined ? isPromptDismissed : false}
+                  showFeedback={showPrompt}
+                  hideFeedback={dismissPrompt}
+                />
+              </ButtonBar>
+            </ErrorBoundary>
+          }
+        >
+          {promptLoading || promptError || isPromptDismissed ? (
```

Comment:
should we have some different states for loading & error? could use `<LoadingIndicator>` or `<LoadingError>`

Reply:
I'm thinking of defaulting to displaying the feedback in either of these cases. So if the prompt API is having problems, users will still be able to see their feedback.

Reply:
that makes sense to me